src/main/java/edu/ie3/datamodel/utils/GridAndGeoUtils.java

```diff
@@ -174,4 +177,25 @@ private static Point buildSafePoint(Point p1) {
         .map(coordinate -> new CoordinateDistance(baseCoordinate, coordinate))
         .collect(Collectors.toCollection(TreeSet::new));
   }
+
+  /**
+   * Calculates the total length of a LineString through building the sum of the distances between
+   * all points of LineString using {@link #calcHaversine(double, double, double, double)}
+   *
+   * @param lineString the line string which length shall be calculated
+   * @return The total length of the line string
+   */
+  public static ComparableQuantity<Length> totalLengthOfLineString(LineString lineString) {
```

Comment:
From my point of view it is a good candidate to be moved to `GeoUtils`, as the parameter `LineString` somewhat is general. Please hand in an issue in PowerSystemUtils to add this method (including tests) and in PowerSystemDataModel to remove this one.

Reply:
done :)